db/migrate/20230926194555_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.references :author, foreign_key: { to_table: :users }
+      t.integer :comments_counter, default: 0
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] [REQUIRED] Kindly note that the migration file for creating Posts table needs a column for the LikesCounter. If you check the entity relationship diagram provided for this task, you will see the column of the LikesCounter on the Post table. Kindly implement this before asking for another review.
``` ruby
t.integer :likes_counter
```